scripts/reel-render-smoke.sh

```
@@ -0,0 +1,49 @@
+echo "═══════════════════════════════════════"
+echo "  naksha — reel render smoke"
+echo "═══════════════════════════════════════"
+
+command -v node >/dev/null 2>&1 || { echo "⏭  SKIP: node not installed"; exit 0; }
+command -v ffmpeg >/dev/null 2>&1 || { echo "⏭  SKIP: ffmpeg not installed"; exit 0; }
+[ -d "$PKG/node_modules" ] || { echo "⏭  SKIP: packages/naksha-reel deps not installed (npm install to enable)"; exit 0; }
+
+# 1. unit tests for the normalize/gate logic (no render needed, always run when deps present)
+echo "▸ unit: normalize + min-beat gate"
+( cd "$PKG" && node --test test/*.test.mjs >/tmp/reel-unit.log 2>&1 ) \
+  && echo "✅ normalize unit tests passed" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)